docs/gatsby-node.js

```diff
@@ -180,13 +182,19 @@ exports.createPages = async ({ graphql, actions, reporter }) => {
     }
   `);
 
-  await createOverviewPages(page => {
-    createPage({
-      path: page.slug,
-      component: path.join(__dirname, "src/templates/Overview.tsx"),
-      context: { ...page },
-    });
-  });
+  const overviewPages = await getOverviewPages();
+  await Promise.all(
+    overviewPages.map(async page => {
+      const trail = await getDocumentTrail(cache, page.slug);
+      const overviewPath = path.join(__dirname, "src/templates/Overview.tsx");
+      createPage({
+        path: page.slug,
+        component:
+          process.env.NODE_ENV === "test" ? path.relative(ROOT, overviewPath) : overviewPath,
+        context: { ...page, trail },
```

Comment:
In order to programmatically create pages in Gatsby a lot of this stuff had to be computed in advance. We can probably fetch some of this data via GraphQL, which I think would split the logic somewhat, and we will because we need it for other stuff, I'd postpone this as a separate effort.